### ISSUE TITLE
car key

### ISSUE THREAD

**Issue by UnamSanctman** [sharebrained/portapack-hackrf#185]
does the portapack hackrf have a feature where it's possible to clone a car key (not unlock signal) but the key itself, cause some cars require the key to be in the car for the car to be able to start

**Comment by Molderfolder** [comment]
Yes it is possible, but most cars that use the key nearby function also use bluetooth le
